filebeat/module/apache2/access/_meta/fields.yml

```diff
@@ -8,6 +8,16 @@
       description: >
         Client IP address or hostname.
 
+    - name: ssl_protocol
```

Comment:
Could we make this ssl.protocol and ssl.cipher?

Reply:
Done. Thanks for pointing this out.